benchexec/runexecutor.py

```diff
@@ -656,7 +663,7 @@ def _setup_ulimit_time_limit(self, hardtimelimit, cgroups):
     def execute_run(self, args, output_filename, stdin=None,
                    hardtimelimit=None, softtimelimit=None, walltimelimit=None,
                    cores=None, memlimit=None, memory_nodes=None,
-                   environments={}, workingDir=None, maxLogfileSize=None,
+                   environments={}, energy_measurement=None, workingDir=None, maxLogfileSize=None,
```

Comment:
Why is `energy_measurement` a parameter instead of just an implementation detail of `RunExecutor`?